cmd/topology.go

```diff
@@ -58,78 +74,208 @@ func runTopology(cmd *cobra.Command, args []string) error {
 	topologyKey := client.ObjectKey{Name: "topology", Namespace: topologyNS}
 	topologyConfigMap := &corev1.ConfigMap{}
 	err = k8sClient.Get(ctx, topologyKey, topologyConfigMap)
-	logf.Log.V(1).Info("reading topology", "object", client.ObjectKeyFromObject(topologyConfigMap), "error", err)
+	logf.Log.V(1).Info("Reading topology ConfigMap", "object", topologyKey, "error", err)
 	if err != nil {
 		return err
 	}
 
-	if topologyDOTOutputFile != "" {
-		fDot, err := os.Create(topologyDOTOutputFile)
+	updateCh := make(chan struct{}, 1)
+
+	var mgr manager.Manager
+	if watchFlag {
```

Comment:
TODO: it would be nice if we could neuter all the `INFO` level logging from controller runtime